ls1intum/Artemis
# `Programming exercises`: Fix programming exercise timeline overflow

### Checklist
#### General



### Motivation and Context


Currently, the programming exercise timeline tooltips are not readable/usable anymore due to an overflow issue (see recording below)

*Tested in `Chromium: 105.0.5195.127`*

### Description

Removing the `overflow-x: scroll` styling and changing the tooltip position of the last timeline entry to the left fixes this issue.

### Steps for Testing

Prerequisites:
- 1 Instructor

1. Log in to Artemis
2. Navigate to Course Administration
3. Create a new programming exercise
4. Check if the tooltips in the timeline are readable

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage


| Class/File | Branch | Line |
|------------|-------:|-----:|


### Screenshots



#### Before

https://user-images.githubusercontent.com/45761921/191691869-5834542f-c3a9-425e-8b34-49245dc2bff5.mp4

#### After


https://user-images.githubusercontent.com/45761921/191699152-92978ceb-355b-4126-bd4c-546c4fa6dd12.mp4
